pandas/tests/extension/base/interface.py

```diff
@@ -97,6 +97,10 @@ def test_isna_extension_array(self, data_missing):
 
             assert na.dtype._is_boolean
 
+    def test_copy_does_not_share_data(self, data):
+        result = data.copy()
+        assert not tm.shares_memory(data, result)
```

Comment:
This helper method will raise an error for external EAs, I think? In that case it might not be super handy to add this test here

Reply:
agreed, will give some thought to an alternative.